cmd/livegrep-fetch-reindex/main.go

```diff
@@ -26,8 +28,17 @@ var (
 	flagReloadBackend = flag.String("reload-backend", "", "Backend to send a Reload RPC to")
 	flagNumWorkers    = flag.Int("num-workers", 8, "Number of workers used to update repositories")
 	flagNoIndex       = flag.Bool("no-index", false, "Skip indexing after fetching")
+	flagMetricsPath   = flag.String("metrics-out", "", "File that indexing metrics should be sent to")
+	flagStatsdOn      = flag.Bool("send-metrics-to-statsd", false, "Send indexing metrics to StatsD")
+	flagStatsdAddr    = flag.String("statsd-address", "", "address URI of statsd listener for metrics export")
+	flagStatsdPrefix  = flag.String("statsd-prefix", "", "optional prefix to apply to all metrics")
+	flagUpdateHead    = flag.Bool("update-head", true, "update the local HEAD ref if it is different than remote. Done when updating already cloned repos.")
```

Comment:
Let's just do this unconditionally, I think the new behavior is just correct and there's no need to add the flag unless there are known sharp edges.

Reply:
Sounds good. I haven't run into any edges so far, will send a patch if I do.